hack/create-user-workspace.sh

```diff
@@ -23,13 +24,13 @@ then
   export KUBECONFIG=${KCP_KUBECONFIG}
 fi
 
-echo "Accessing the home workspace:"
-kubectl ws '~'
-
 if [ "${ROOT_WORKSPACE}" == "~" ]; then
   ROOT_WORKSPACE=$(kubectl ws . --short)
```

Comment:
then we need to change this to:
```suggestion
  ROOT_WORKSPACE=$(kubectl ws '~' --short)
```